examples/ota-requestor-app/esp32/README.md

```diff
@@ -0,0 +1,91 @@
+# CHIP ESP32 OTA Requestor Example
+
+A prototype application that demonstrates OTA Requestor capabilities.
+
+## Prerequisite
+
+Before moving ahead, make sure you have
+[OTA Provider](../../ota-provider-app/esp32) is commissioned and running.
+
+## Supported Devices
+
+-   This example supports ESP32 and ESP32C3. For details please check
+    [here](https://github.com/shubhamdp/connectedhomeip/tree/shubhamdp-patch-1/examples/all-clusters-app/esp32#supported-devices).
+
+## Building the Example Application
+
+-   If you are building for the first time please check
+    [Building the Example Application](https://github.com/shubhamdp/connectedhomeip/tree/shubhamdp-patch-1/examples/all-clusters-app/esp32#building-the-example-application)
+    guide.
+-   Otherwise, `idf.py build` works!
+
+## Flashing the Example Application
+
+```
+idf.py -p <OTARequestorSerialPort> flash monitor
+```
+
+## Commissioning over BLE using chip-tool
+
+-   Please build the standalone chip-tool as described [here](../../chip-tool)
+-   Commissioning the OTA Requestor
+
+```
+./out/debug/chip-tool pairing ble-wifi 12346 <ssid> <passphrase> 0 20202021 3840
+```
+
+## Query for an OTA Image
+
+After commissioning is successful, query for OTA image. Head over to ESP32
+console and fire the following command. This command creates a CASE session with
+OTA Provider and start the OTA image transfer using BDX protocol.
+
+```
+esp32> QueryImage <OTAProviderIpAddress> <OTAProviderNodeId>
+```
+
```

Comment:
Mention about it takes some time and when it is complete it will print some specific log on the console?

Reply:
Added the suggestion.